bigbluebutton-html5/imports/api/meetings/server/methods/setFitToWidth.js

```diff
@@ -0,0 +1,32 @@
+import Meetings from '/imports/api/meetings';
+import { extractCredentials } from '/imports/api/common/server/helpers';
+import { check } from 'meteor/check';
+import Logger from '/imports/startup/server/logger';
+
+export default function setFitToWidth(fitToWidth) {
+
+  try {
+    const { meetingId, requesterUserId } = extractCredentials(this.userId);
+
+    check(meetingId, String);
+    check(fitToWidth, Boolean);
+
+    const selector = {
+      meetingId,
+    };
+
+    const modifier = {
+      $set: {
+        fitToWidth,
+      }
+    }
+
+    const { insertedId } = Meetings.upsert(selector, modifier);
```

Comment:
Note - here a new `Meeting` property is being added in MongoDB and it does not reach akka-apps. I'll need to think about this some more.

Reply:
Correct. I did it so on purpose, wondering (still) what is the best way to implement. 
We can also send this property to akka-apps and propagate it to other users/viewers, which can potentially become presenters. Actually I tried to do so initially, but then I realised that we do not need to send it to other viewers; we just want BBB to "remember" it just for the presenter. I also felt that the fitToWidth property is rather a personal setting than a meeting-wide.
Of course this PR is just an idea, you can decide the design. Thanks for the comment!

Reply:
Note: just to remember the fitToWidth property during the session, we can use the meteor/session. I don't know which is better.